package.json

```diff
@@ -1,9 +1,12 @@
 {
   "scripts": {
     "test": "",
-    "build": "node-sass --include-path node_modules static/sass --output static/css && postcss --use autoprefixer --replace static/css/*.css",
+    "build": "yarn run build-js && yarn run build-css",
+    "copy-css": "cp node_modules/cookie-policy/build/css/cookie-policy.css static/css/modules && cp node_modules/global-nav/build/css/main.css static/css/modules",
```

Comment:
Did you consider doing this with a symlink - ([like www.ubuntu.com](https://github.com/canonical-websites/www.ubuntu.com/tree/master/static)) . I prefer the symlink solution because it makes it clear that there's supposed to be a file there, and that it's expected to come from node_modules, simply from looking in the filesystem.

Reply:
Oh I was too late. That's a shame. @tbille @WillMoggridge what do you think anyway?

Reply:
I think this would have been a better idea